components/brave_news/browser/resources/internals.tsx

```diff
@@ -21,14 +21,24 @@ const Grid = styled.div`
   gap: 8px;
 `
 
+const Blur = styled.div`
+  --edges: -128px;
+
+  z-index: -1;
+  position: fixed;
+  inset: var(--edges);
+  filter: blur(64px);
+  background: rgba(255, 255, 255, 0.7);
+`
 
 function App() {
-  return (
+  return <>
+    <Blur />
```

Comment:
Can the blur be achieved without an extra element via a backrop-filter on `Grid`? Or a pseudo element if there's some reason it needs another element?

Reply:
Completely forgot backdrop-filter was a thing!